File: src/main/java/nodenet/proxy/ClientProxy.java
package nodenet.proxy;

import net.minecraftforge.common.MinecraftForge;

import net.minecraft.client.Minecraft;
import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.util.math.Vec3d;

import nodenet.Conf;
import nodenet.register.ItemRegister;
import nodenet.register.NodeRegister;
import nodenet.render.ShaderHelper;

public class ClientProxy extends ModProxy {

	@Override
	public EntityPlayer getPlayerLooking(Vec3d start, Vec3d end) {
		return Minecraft.getMinecraft().thePlayer;
	}

	@Override
	public void reloadConfigs() {}

	@Override
	public void preInit() {
		ItemRegister.initRender();
		NodeRegister.initRender();
		
		MinecraftForge.EVENT_BUS.register(new Conf());
		
		ShaderHelper.initShaders();
	}

}

File: src/main/java/com/thecodewarrior/nodenet/client/ClientProxy.kt
package com.thecodewarrior.nodenet.client

import com.thecodewarrior.nodenet.client.render.NodeWorldRenderer
import com.thecodewarrior.nodenet.common.CommonProxy
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent

class ClientProxy: CommonProxy() {
    override fun pre(event: FMLPreInitializationEvent) {
        super.pre(event)
        NodeWorldRenderer
    }
}